worlds/celeste64/__init__.py

```diff
@@ -0,0 +1,93 @@
+from typing import List
+
+from BaseClasses import ItemClassification, Region, Tutorial
+from worlds.AutoWorld import WebWorld, World
+from .Items import Celeste64Item, item_data_table, item_table
+from .Locations import Celeste64Location, location_data_table, location_table
+from .Names import ItemName
+from .Options import Celeste64Options
+
+
+class Celeste64WebWorld(WebWorld):
+    theme = "ice"
+
+    setup_en = Tutorial(
+        tutorial_name="Start Guide",
+        description="A guide to playing Celeste 64 in Archipelago.",
+        language="English",
+        file_name="guide_en.md",
+        link="guide/en",
+        authors=["PoryGone"]
+    )
+
+    tutorials = [setup_en]
+
+
+class Celeste64World(World):
+    """Relive the magic of Celeste Mountain alongside Madeline in this small, heartfelt 3D platformer.
+    Created in a week(ish) by the Celeste team to celebrate the game’s sixth anniversary 🍓✨"""
+
+    game = "Celeste 64"
+    data_version = 0
```

Comment:
```suggestion
```
Even if you don't want to get rid of it, it probably should be incremented to 1 at least.

Reply:
the base World definition already defines it at 0 so it's not necessary to define it on your subclass